### ISSUE TITLE
Mobs should have some way of fighting turrets

### ISSUE THREAD

**Issue by somebodyandnobody** [OpenModularTurretsTeam/OpenModularTurrets#531]
#### Issue Description:

The turrets can take down the strongest mobs without much trouble as long as they have enough RF. Mobs should have some way of destroying them or deactivating them.

#### What happens:

Mobs will just sit there and take damage. They don't even run away.

**Comment by MartijnPeters1998** [comment]
Seems like a  good idea, but only hostile mobs can attack turrets with that feature right?

**Comment by somebodyandnobody** [comment]
Yes, of course. Animals attacking a turret would be downright strange.
